ref/general/requestTiming.adoc

```diff
@@ -0,0 +1,71 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: The request timing feature automatically detects slow requests and logs the information when the request is completed.
+:page-categories: MicroProfile
+:seo-title: The request timing feature automatically detects slow and hung requests
+:seo-description: The request timing feature automatically detects slow and hung requests and logs the information when the request is completed.
+= Slow and hung request detection
+
+Slow or hung requests can happen, and your application needs a continuous monitoring tool to detect the cause.
+The request timing feature automatically detects slow  and hung requests and logs the information when the request is completed.
+
+The `requestTiming` feature monitors requests with respect to time and automatically detects and diagnoses slow and hung requests.
+In addition to the previous capability of detecting the slow request, the updated feature now also logs when the request is completed, thus improving the monitoring efficiency.
+The users can view the information about a completed slow request with the request id and time of completion.
```

Comment:
maybe don't switch to "users" here- as there is some ambiguity about end user of an app vs developer. Stay with "You can view..."

Does this sentence mean that you need to know the request id and time of completion to view info about a request? It's not clear

Reply:
The Request Timing feature will automatically detect and provide the information about the request id and time of completion. Updated the doc.